setup.py

```diff
@@ -7,7 +7,7 @@
     install_requires=[
         'astropy~=4.2',
         'ctapipe~=0.10.0',
```

Comment:
what about `ctapipe>=0.10.4` ?

Reply:
~= means >= 0.10.0,<0.11, so any 0.10.x version is supported. I think this is what we want. No need to require >= 0.10.4.